### PR TITLE
remove function calls to SVD

### LINKED ISSUE #414: Use look-up table for "random" data in test functions
Since it is cumbersome to come up with random data in both MATLAB and Octave, it is proposed to drop the use of `rand()`, ~~`magic()`~~ and `svd()` inside `ACID.m`. Instead, pseudo random data from a look-up table should be used.
See commit https://github.com/PeterPablo/matlab2tikz/commit/734bed7332ecf04a38503e660740b9f161ecd1c2 of pull request https://github.com/nschloe/matlab2tikz/pull/413 for discussion.

### PR DESCRIPTION
**Pull request by PeterPablo**
* introduce sub function that returns a small data table
* all svd calls are replaced
* this should hopefully partially resolve issues, observed e.g. in #590 and #604

Should partially fix #414.

### CODE REVIEW

**Comment by PeterPablo** [comment]
Failing tests:

```
--- mine/test25-converted.tex
+++ (clipboard)
@@ -12,7 +12,7 @@
 \begin{tikzpicture}
 
 \begin{axis}[%
-width=0.95092\figurewidth,
+width=0.75\figurewidth,
 height=\figureheight,
 at={(0\figurewidth,0\figureheight)},
 scale only axis,
```
Unfortunately `width` is determined differently. I raised this issue in #https://github.com/matlab2tikz/matlab2tikz/issues/552#issuecomment-76918634.

**Comment by PeterPablo** [comment]
How should we proceed?

**Comment by okomarov** [comment]
I reckon there are two things to do first:
* Define width and height explicitly as in (but without the float format change) https://github.com/PeterPablo/matlab2tikz/commit/9e31d1bf57d9ca632fc935e8b46752476f5a9608
* Remove instances of `hist()`. TMW is deprecating it and one reason is [inconsistent behaviour](http://uk.mathworks.com/help/matlab/creating_plots/replace-discouraged-instances-of-hist-and-histc.html) (wathever that means) 

Basically, this PR will then address partially #552 and #414.

**Comment by PeterPablo** [comment]
Sounds reasonable. Good point especially about removing `hist()`.
In order to introduce explicit `width` and `height` we have to adapt https://github.com/PeterPablo/matlab2tikz/commit/9e31d1bf57d9ca632fc935e8b46752476f5a9608 to not pass latex variables, but rather directly supply dimensions -- otherwise the width still contains a calculated value.
So, should I first focus on this PR?

**Comment by okomarov** [comment]
I think so, but I am afraid is gonna be hard :sweat_smile: now that I realize what it entails.

**Comment by PeterPablo** [comment]
:-/ A lot more hash calculation fun... I really hope it is worth the effort and we are rewarded by testsuite stability. Actually I would not remove calls to `hist()` here, but rather do this as a second PR, to not introduce too many changes in one go. I will though (re)label them as *unreliable* and link to your comment.
Furthermore I will adapt the above linked commit of mine to incorporate fixed dimensions. * crossing fingers this does not introduce new issues *

**Comment by PeterPablo** [comment]
Oh, I'll wait for a short comment of @egeerardyn, so we agree on course of action.

**Comment by egeerardyn** [comment]
I'm on holiday until Sunday, so I will try to have a look early next week.

**Comment by egeerardyn** [comment]
With regards to `hist`, I'm on the one hand very happy that TMW is deprecating it (it has caused a lot of indeed inconsistent behavior, also in our codebase: the internals  of `hist` are a terrible mess).

Anyhow, I think we should keep our limited `hist` support at least for one more year. Most people I know have not migrated to R2014b, let alone R2015a (where `hist` has been deprecated), so we still need to keep their condition in mind. E.g. here in my lab it is unfeasible to migrate completely to R2014b since some of our own toolboxes are still not compatible and I can imagine that other labs have similar issues.

When (not if) we remove `hist`, it should happen in a different PR, regardless of the actual time when we do this.

Regarding the axes dimensions: I think that we always calculate their size, as long as it is specified in a valid LaTeX format, so that course of action is probably not helping anything. But let's discuss this in a separate issue report and keep this one focussed on the current changes.

**Comment by PeterPablo** [comment]
Let's keep this PR's scope limited to `SVD`. `width` and `hist()` will be dealt with separately.

I restored the `unreliable` status of two tests, though added a comment about why this test is unreliable.
I removed the `unreliable` flag of `bars` (= `ACID(34)`). Please test, if hashes match for you!

If no further issues arise, I consider this ready.

**Comment by PeterPablo** [comment]
Please let's first get #590 in. I will then merge `develop` here and update the hashes. Afterwards somebody has to verify `ACID(34)` and this is then ready to go in.

**Comment by PeterPablo** [comment]
I rebased on `HEAD`, updated the hashes and marked one more test unreliable and linked to the issue/discussion. Please test `ACID(34)` on Matlab R2014(a|b) and merge, as soon as Travis ran successfully.
